ChicagoLandmarks/ChicagoLandmarks/DataManager.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  DataManager.swift
+//  ChicagoLandmarks
+//
+
+import Foundation
+import MapKit
+
+public class DataManager {
+  
+    // MARK: - Singleton Stuff
+    public static let sharedInstance = DataManager()
+  
+    //This prevents others from using the default '()' initializer
+    fileprivate init() {}
+
+    // Your code (these are just example functions, implement what you need)
+    func loadAnnotationFromPlist() -> [Place]{
+        let pathBundle = Bundle.main.path(forResource: "Data", ofType: ".plist")
+        let root = NSDictionary(contentsOfFile: pathBundle!)
+        let places:[NSDictionary] = root?["places"] as! Array<NSDictionary>
+        var result:[Place] = []
+        for item in places {
+            let place = Place()
+            place.name = item["name"] as? String
+            place.longDescription = item["description"] as? String
+            let lat = item["lat"] as! Double
+            let long = item["long"] as! Double
+            place.coordinate = CLLocationCoordinate2DMake(lat, long)
+            result.append(place)
+        }
+        return result
+    }
+    
+    func loadRegionFromPlist() -> [Any]{
+        let pathBundle = Bundle.main.path(forResource: "Data", ofType: ".plist")
+        let root = NSDictionary(contentsOfFile: pathBundle!)
+        let result:[Any] = root?["region"] as! Array<Any>
+        return result
+    }
+    
+    func saveFavorite(favorite:String) {
+        let defaults = UserDefaults.standard
```

Comment:
Since you use this `defaults` multiple times, make it a property of the class instead to reduce redundancy.